app/config-adjusters.ts

```diff
@@ -132,6 +136,7 @@ type AreaAdjusters = BaseAdjusters<AreaConfig> & {
   fields: {
     x: { componentId: FieldAdjuster<AreaConfig, string> };
     y: { componentId: FieldAdjuster<AreaConfig, string> };
+    color: FieldAdjuster<ColumnConfig, ColorField>;
```

Comment:
@noahonyejese it looks like the type of chart config is wrong in a few places (`ColumnConfig` is used in area, line, bar and column adjusters) 👓